build/jobs/run-tests.yml

```diff
@@ -88,18 +88,18 @@ jobs:
       ScriptType: inlineScript
       Inline: |
         $secrets = Get-AzKeyVaultSecret -VaultName $(DeploymentEnvironmentName)-ts
-
+        Write-Host "From TS"
         foreach($secret in $secrets)
         {
             $environmentVariableName = $secret.Name.Replace("--","_")
 
             $secretValue = Get-AzKeyVaultSecret -VaultName $(DeploymentEnvironmentName)-ts -Name $secret.Name
+            Write-Host "Overwriting $environmentVariableName with value $secretValue"
```

Comment:
Do we need to log this here? Or is it just for debugging purposes?

Reply:
Good catch, removed